### PR TITLE
Expand cluster

### PR DESCRIPTION
**Pull request by Praburaj-Krishnan**

### CODE REVIEW

File: .settings/launch.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (kmkanagaraj) [inline .settings/launch.json]: Remove this file from the commit

File: app/components/clusters/cluster-expand.ts [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        this.disks = [];
+        this.osds = [];
+        this.clusterID =  this.routeParamsSvc['id'];
+        
+        this.clusterService.get(this.clusterID).then(this.clusterGetCallBack);
+        this.serverService.getDiscoveredHosts().then(this.discoveredHostCallBack);
+        this.serverService.getFreeHosts().then(this.freeHostCallBack);
+    }
+
+    public clusterGetCallBack = (cluster: any) => {
```
Comment (kmkanagaraj) [inline app/components/clusters/cluster-expand.ts]: rename it to loadCluster. Change this to a normal function and use arrow function above

File: app/components/clusters/cluster-expand.ts [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+        this.cluster = cluster;
+        this.name = cluster.cluster_name;
+        this.logService.info("Name: " + this.name);
+    }
+
+    public discoveredHostCallBack = (freeHosts: any) => {
```
Comment (kmkanagaraj) [inline app/components/clusters/cluster-expand.ts]: rename it to loadDiscoveredHosts, and move the arrow to constructor

File: app/components/clusters/cluster-expand.ts [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+            () => {
+                this.newHost.cautionMessage = "Error!.";
+                this.newHost.errorMessage = "Could not resolve the hostname";
+            },
+            (fingerPrint) => {
+                host.fingerprint = fingerPrint;
+            });
+    }
+
+    public freeHostCallBack = (freeHosts: any) => {
```
Comment (kmkanagaraj) [inline app/components/clusters/cluster-expand.ts]: Rename to loadFreeHosts

File: .settings/launch.json (hunk 1 above)
Comment (Praburaj-Krishnan) [inline .settings/launch.json]: But I don't see any file like that in my directory

Comment (kmkanagaraj) [inline .settings/launch.json]: .settings/launch.json

Comment (Praburaj-Krishnan) [inline .settings/launch.json]: Its in hidden folder just now saw. Deleted the file.

**Pull request merged**